ChatQnA/kubernetes/intel/cpu/xeon/manifest/chatqna.yaml

```diff
@@ -238,7 +211,7 @@ data:
         }
 
         location / {
-            proxy_pass http://chatqna-chatqna-ui:5174;
```

Comment:
Why do we need to change this port number? This is the port of k8s chatqna-chatqna-ui service. End user should always access the nginx service, not directly access the ui service.